next.config.js

```diff
@@ -1,8 +1,11 @@
 module.exports = {
+  experimental: {
+    outputStandalone: true,
+  },
   images: {
-    domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS.split(",")
-      .map((d) => d.trim())
-      .filter((d) => d),
+    domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS?.split(",")
+      ?.filter((d) => d.trim())
+      ?.map((d) => d.trim()),
```

Comment:
```suggestion
      ?.map((d) => d.trim())
      ?.filter((d) => d),
```
Must transform first (map) and then filter out those that are not empty... `TwoopsTrackerUI` code is wrong as well.